staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -330,6 +330,27 @@ if (Test-Path $nvidiaInstallLogFolder) {
   }
 }
 
+# Collect kubectl information
+Write-Host "Collecting kubectl information"
+function kubectl { c:\k\kubectl.exe --kubeconfig c:\k\config $args }
+kubectl get nodes -o wide > "$ENV:TEMP\kubectl-get-nodes-$($timeStamp).log"
```

Comment:
We should handle the exception if the command fails, for example, it cannot connect to the api server.

Reply:
Added try-catch, the log is
![image](https://github.com/Azure/AgentBaker/assets/62196586/35d87636-d504-4da3-8e12-4106173d4aa0)